stdlib/runtime/wasi.gr

```diff
@@ -176,17 +323,21 @@ let _ENOTCAPABLE = 76n
 
 exception SystemError(Number)
 
-let stringOfSystemError = (code) => {
+let stringOfSystemError = code => {
   let n = WasmI32.shrS(WasmI32.fromGrain(code), 1n)
   let (==) = WasmI32.eq
   match (n) {
-    n when n == _ESUCCESS => "SystemError: No error occurred. System call completed successfully.",
+    n when (
+      n == _ESUCCESS
+    ) => "SystemError: No error occurred. System call completed successfully.",
```

Comment:
@marcusroberts this is weird. Why does it wrap the `when` before wrapping the statement after the arrow?

Reply:
Fixed by breaking after the =>